modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -34,3 +34,25 @@ $ oc describe csr <csr_name> <1>
 ----
 $ oc adm certificate approve <csr_name>
 ----
+
+In case your load balancer certs are also expired you will need to talk to one of the masters directly.
+
+. Setup an ssh tunnel
++
+[source,terminal]
+----
+$ ssh -i <master_private_key> -N -L 42443:localhost:6443 -o ProxyCommand='ssh -i <bastion_private_key> -W %h:%p <bastion_user>@<bastion>' core@<master>
+----
+<1> `<master_private_key>` is the ssh key to to use to connect to your master.
+<2> `<bastion_private_key>` is the ssh key to to use to connect to your bastion.
```

Comment:
We specifically don't actually talk about having a bastion in any of the DR docs. I think that we purposefully don't cover specifics on SSHing, so I'm thinking that might apply here as well.

For example, this says just "Establish SSH connectivity..": https://docs.openshift.com/container-platform/4.6/backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

Would it be sufficient to just say to set up the SSH tunnel without giving the specific command/parameters? We can chat offline about this if you'd like.